Tasks/DropValidatorV0/main.ts

```diff
@@ -0,0 +1,62 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as telemetry from 'azure-pipelines-tasks-utility-common/telemetry';
+import * as fs from 'fs';
+
+async function run() {
+    try {
+        const dropValidatorArgs = ['BuildDropPath', 'ManifestPath', 'OutputPath', 'Verbosity', 'ConfigFilePath', 'RootPathFilter'];
```

Comment:
Do we validate the OutputPath inside the dropvalidator code? I'm just curious if everything will succeed except trying to write the results because the OutputPath is incorrect.

Reply:
The value is a relative path from the build agent run folder, and it's writable. We also check on the dropvalidator.exe to see if this location is not writable and log an error accordingly.